list/harvesterhci.io.cloudtemplate.vue

```diff
@@ -48,7 +48,7 @@ export default {
       ];
     },
 
-    filterdRows() {
+    filteredRows() {
```

Comment:
api?

Reply:
Computed prop for this component, this change is safe